Update troubleshooting docs (#21538)
gravitational/teleport: docs/pages/management/admin/troubleshooting.mdx

To diagnose problems, you can configure the `teleport` process to run with
verbose logging enabled by passing it the `-d` flag. `teleport` will write logs

> to stderr. 

Reviewer: Remove trailing space.

> Alternatively to the `-d` flag you can set the log level within the Teleport configuration file.

Reviewer: ```suggestion
Alternatively, you can set the log severity to `DEBUG` in the configuration file.
```

Reviewer: Disregard this one, Alex made a similar suggestion and his is better.

> Alternatively to the `-d` flag you can set the log level within the Teleport configuration file.
> ```yaml
> teleport:
>   log:
>     severity: DEBUG
> ```
> 
> Restart the `teleport` process to apply the modified log level. Logs will resemble the following
> (these logs were printed while joining a Node to a cluster, then terminating the `teleport` process on the Node):

Reviewer 2: Suggestions for formatting, brevity, and clarity:

````suggestion
Alternatively, you can set the log level from the Teleport configuration file:

```yaml
teleport:
  log:
    severity: DEBUG
```

Restart the `teleport` process to apply the modified log level. Logs will resemble
the following (these logs were printed while joining a Node to a cluster, then
terminating the `teleport` process on the Node):
````

verbose logging enabled by passing it the `-d` flag. `teleport` will write logs
to stderr.

Alternatively, you can set the log level from the Teleport configuration file:

```yaml
teleport:
  log:
    severity: DEBUG
```

> Restart the `teleport` process to apply the modified log level. Logs will resemble
> the following (these logs were printed while joining a Node to a cluster, then
> terminating the `teleport` process on the Node):

Reviewer 3: ```suggestion
Restart the `teleport` process to apply the modified log level. Logs will resemble
the following (these logs were printed while joining a server to a cluster, then
terminating the `teleport` process on the server):
```
Since we're not using the "Node" concept anymore
